bin/test_travis.sh

```diff
@@ -126,7 +132,7 @@ else
     if [[ "${WITH_MPFR}" == "yes" ]] || [[ "${WITH_MPC}" == "yes" ]] || [[ "${WITH_ARB}" == "yes" ]]; then
         extra_libs="$extra_libs -lmpfr"
     fi
-    ${CXX} -std=c++0x -I$our_install_dir/include/ -L$our_install_dir/lib test_basic.cpp -lsymengine -lteuchos $extra_libs -lgmpxx -lgmp
+    ${CXX} -std=c++0x -I$our_install_dir/include/ -I$SOURCE_DIR/symengine/catch -L$our_install_dir/lib test_basic.cpp -L$BUILD_DIR/symengine/catch -lcatch -lsymengine -lteuchos $extra_libs -lgmpxx -lgmp
```

Comment:
Yes, I think the last few patches is the way to go.

Sent from my mobile phone.
On Jun 30, 2015 2:05 AM, "Sumith" <notifications@github.com> wrote:

> In bin/test_travis.sh
> <https://github.com/sympy/symengine/pull/484#discussion_r33548944>:
>
> > @@ -126,7 +132,7 @@ else
> >      if [[ "${WITH_MPFR}" == "yes" ]] || [[ "${WITH_MPC}" == "yes" ]] || [[ "${WITH_ARB}" == "yes" ]]; then
> >          extra_libs="$extra_libs -lmpfr"
> >      fi
> > -    ${CXX} -std=c++0x -I$our_install_dir/include/ -L$our_install_dir/lib test_basic.cpp -lsymengine -lteuchos $extra_libs -lgmpxx -lgmp
> > +    ${CXX} -std=c++0x -I$our_install_dir/include/ -I$SOURCE_DIR/symengine/catch -L$our_install_dir/lib test_basic.cpp -L$BUILD_DIR/symengine/catch -lcatch -lsymengine -lteuchos $extra_libs -lgmpxx -lgmp
>
> Let me try that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/484/files#r33548944>.
>